teps/0092-scheduling-timeout.md

```diff
@@ -45,8 +49,22 @@ authors:
 
 ## Summary
 
-Add runtime configuration options to allow setting a timeout for `TaskRuns` which applies before the `TaskRun's`
-underlying pod is scheduled, and only start our existing timeout counters after this timeout has passed, if specified.
+This TEP proposed adding runtime configuration options to allow setting a timeout for `TaskRuns` which would have 
+applied before the `TaskRun's` underlying `Pod` is scheduled, then we would start our existing timeout counters after
+this new timeout had passed, if specified.
+
+However, we decided to pursue supporting queueing directly instead of adding a queueing/scheduling timeout because:
```

Comment:
The proposal was to support specifying `scheduling` time at the pipelineRun level. This scheduling time applies to each `pipelineTasks` of that pipeline (including `tasks` and `finally`).

"At the start of each Task being executed as a TaskRun, the `scheduling` timeout counter would begin, considering the `timeout` to have started from the creation time of the TaskRun in etcd".

Reply:
yes, it was at [pipelinerun](https://github.com/tektoncd/community/blob/main/teps/0092-scheduling-timeout.md#pipelinerun) and [taskrun](https://github.com/tektoncd/community/blob/main/teps/0092-scheduling-timeout.md#taskrun) levels